qiime2/sdk/plugin_manager.py

```diff
@@ -100,11 +101,28 @@ def _integrate_plugin(self, plugin):
 
             self.formats[name] = record
         self.type_formats.extend(plugin.type_formats)
+        self.importable_types.update(plugin.importable_types)
 
     # TODO: Should plugin loading be transactional? i.e. if there's
     # something wrong, the entire plugin fails to load any piece, like a
     # databases rollback/commit
 
+    @property
```

Comment:
`get_semantic_types` should be a method (a callable function on an instance of `PluginManager`, not a property. Please remove this!